format/spec.md

```diff
@@ -685,6 +686,7 @@ A snapshot consists of the following fields:
 | _optional_ | _optional_ | _optional_ | **`schema-id`**              | ID of the table's current schema when the snapshot was created                                                                     |
 |            |            | _optional_ | **`first-row-id`**           | The first `_row_id` assigned to the first row in the first data file in the first manifest, see [Row Lineage](#row-lineage)        |
 |            |            | _optional_ | **`added-rows`**             | Sum of the [`added_rows_count`](#manifest-lists) from all manifests added in this snapshot. Required if [Row Lineage](#row-lineage) is enabled | 
+|            |            | _optional_ | **`key-metadata-id`**        | ID of the manifest list key metadata in the encryption-keys table ([Encryption Keys](#encryption-keys))                            |
```

Comment:
maybe we don't need this addition.. We can use the snapshot-id to reference the snapshot's key-metadata in the encryption-keys table.